cloud_experts_tutorials/cloud-experts-rosa-with-hcp-private-offer-acceptance-and-sharing.adoc

```diff
@@ -0,0 +1,187 @@
++
+.Regular private offer
++
+image::rosa-regular-private-offer.png[]
++
+In Figure 2, you can see a private offer that was created for an AWS account that previously activated {hcp-title} using the public offer, showing the product name and the selected private offer labeled as _Upgrade_, that replaces the currently running contract for {hcp-title}.
++
+.Private offer selection selection screen
++
+image::rosa-private-offer-selection-selection-screen.png[]
+
+. The drop down menu allows selecting between multiple offers, if available. In Figure 3, the previously activated public offer is shown together with the newly provided agreement based offer that is labeled as “Upgrade”.
++
+.Private offer selection dropdown
++
+image::rosa-private-offer-selection-dropdown.png[]
+
+. Figure 4 shows the bottom part of the offer page with the offer details. Check that the offer configuration reflects your expectations. 
++
+[NOTE]
+====
+The contract end date, the number of units included with the offer, and the payment schedule. In this example, 1 cluster and up to 3 nodes utilizing 4 vCPUs are included.
+====
++
+.Private offer details
++
+image::rosa-private-offer-details.png[]
+
+. Optionally, you can link:https://docs.aws.amazon.com/marketplace/latest/buyerguide/buyer-purchase-orders.html[add your own purchase order (PO) number] to the subscription that is being purchased, so it will be included on your subsequent AWS invoices. Also, check the _Additional usage fees_ that are charged for any usage above the scope of the _New offer configuration details_.
++
+[NOTE]
+====
+Private offers have several available configurations. It is possible that the private offer you are accepting is set up with a fixed future start date. If you do not have another active {hcp-title} subscription at the time of accepting the private offer, a public offer or an older private offer entitlement, accept the private offer itself and continue with the account linking and cluster deployment steps after the specified service start date. Neither of these steps is possible without an active {hcp-title} entitlement. Service start dates are always reported in the UTC time zone.
+====
+
+. If all the details match your expectations, click the *Create contract* button, shown in Figure 5, for private offers accepted by an AWS account that does not have the {hcp-title} activated yet and is creating the first contract for this service.
++
+.Create contract button
++
+image::rosa-create-contract-button.png[]
+
+. Or click the *Upgrade current contract button* for agreement-based offers as seen in Figures 4 and 6.
++
+.Upgrade contract button
++
+image::rosa-upgrade-contract-button.png[]
+
+. There is one more confirmation modal window shown, similar to the one shown in Figure 7.
```

Comment:
Instead of "There is one more confirmation modal window shown, similar to the one shown in Figure 7.", maybe we could make it more action oriented because it is a step. "Confirm your purchase in the modal shown in Figure 7."